.github/labeler.yml

```diff
@@ -31,3 +31,6 @@ ci:
   - '.github/**'
   - 'dockers/**'
   - '.pre-commit-config.yml'
+
+docs:
```

Comment:
We generally only use the docs label for PRs that only focus on docs
But many PRs do a small docs modification. This change would add it to them.

Do you want this?

Reply:
That is fine, but I was thinking about how to automate it...

Reply:
ok, dropped for now :chipmunk: